backend/src/personnel/dto/get-personnel.dto.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { Length } from 'class-validator';
+import { Region, WorkLocation } from '../../common/enums';
+import { QueryDTO } from '../../query-validation.pipe';
+
+export class GetPersonnelDTO extends QueryDTO {
+  @ApiPropertyOptional({
+    description: 'Name of personnel - can be a substring of first or last name',
+  })
+  @Length(3)
+  name: string;
+
+  @ApiPropertyOptional({
+    description: 'Find only active personnel. If false, this will find all',
+    default: true,
+  })
+  active: boolean;
+
+  @ApiPropertyOptional({
```

Comment:
We will need to include in this endpoint the boolean toggle the "show unavailable"

Reply:
(sorry I realize this is the dto not the endpoint :P)